infoschema/tables.go

```diff
@@ -929,6 +932,11 @@ var tableInspectionSummaryCols = []columnInfo{
 	{name: "MAX_VALUE", tp: mysql.TypeDouble, size: 22, decimal: 6},
 }
 
+var tableInspectionRulesCols = []columnInfo{
+	{name: "NAME", tp: mysql.TypeVarchar, size: 64},
+	{name: "TYPE", tp: mysql.TypeVarchar, size: 64},
```

Comment:
It's better to add a column `COMMENT`, we may add some explanation for the rule. （You can open a new pr to add an explanation to each rule）

Reply:
now I add a column `COMMENT` leaving empty ( I should read docs later to understand those rules..

Reply:
Thanks